web/src/components/pages/dashboard/goals/custom-goal.tsx

```diff
@@ -194,6 +194,7 @@ export default function CustomGoal({
   dashboardLocale: string;
 }) {
   const { history, location } = useRouter();
+  const l10n = useLocalization();
```

Comment:
I think the correct use is:
```
const {l10n} = useLocalization();
```
This is what is exposed.
See: https://github.com/projectfluent/fluent.js/blob/main/fluent-react/src/use_localization.ts